addons/point_of_sale/point_of_sale_view.xml

```diff
@@ -179,16 +179,51 @@
         <menuitem name="Orders" id="menu_point_of_sale" parent="menu_point_root" sequence="10"/>
         <menuitem parent="menu_point_of_sale" id="menu_point_ofsale" action="action_pos_pos_form" sequence="1" groups="group_pos_manager,group_pos_user"/>
 
+        <record id="product_template_form_view" model="ir.ui.view">
+            <field name="name">product.template.form.inherit</field>
+            <field name="model">product.template</field>
+            <field name="inherit_id" ref="product.product_template_form_view"/>
+            <field name="arch" type="xml">
+                <group name="sale_condition" position="after">
+                    <group name="pos" string="Point of Sale">
+                        <field name="available_in_pos"/>
+                        <field name="pos_categ_id" attrs="{'invisible': [('available_in_pos', '=', False)]}"/>
+                        <field name="to_weight" attrs="{'invisible': [('available_in_pos', '=', False)]}"/>
+                    </group>
+                </group>
+            </field>
+        </record>
+        <record id="product_template_tree_view_pos" model="ir.ui.view">
```

Comment:
If you change the xml_id (and name), update it according to the guidelines :) (product_template_view_tree_inherit_pos)